inst/ref/chi_qa.yaml

```diff
@@ -18,6 +18,9 @@ vars:
     upper_bound: float
     se: float
     rse: float
+    comparison_with_kc: varchar(255)
```

Comment:
all the changes look good except for this one. Joie asked me to drop these columns for this year. I'll write her in moment to ask her to update the SharePoint documentation to too.